### PR TITLE
Make the conformance test names unique

### LINKED ISSUE #508: Impossible to view test details in conformance test GUI
Two of the conformance tests have the same name and get the same ID in the web-report meaning that it is impossible to open the second one (clicking on it will only open the first one).

Tests:
https://github.com/opencontainers/distribution-spec/blob/05cf62402d050b09953f4ed9a041666d150d8718/conformance/01_pull_test.go#L174

and 
https://github.com/opencontainers/distribution-spec/blob/05cf62402d050b09953f4ed9a041666d150d8718/conformance/01_pull_test.go#L186

Apart from giving them separate IDs I would suggest giving them different names to simplify distinguishing them.

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Some of the conformance tests have the same name, which doesn't render correctly in the results web page.

Fixes #508.

### CODE REVIEW

**Comment by sudo-bmitch** [comment]
Note to maintainers, please hold off on merging until the 1.1.0 vote completes.

**Review by austinvazquez — APPROVED** [review]

**Review by dmcgowan — APPROVED** [review]

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**
